ansible/host_vars/a-fsn-de.m.voidlinux.org.yml

```diff
@@ -39,3 +39,6 @@ nomad_host_volumes:
   - name: ccache
     path: /hostdir/ccache
     read_only: true
+
+nomad_extra_caps:
+  - sys_admin
```

Comment:
This is very broad.  It should probably be in a group for the hosts that need it with a comment explaining what actions require the use of the top level `CAP_SYS_ADMIN` instead of one of the more granular tokens.